src/js/Slider.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import ReactSlider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+import store from './store'
+import { updateSliderPosition } from './actions'
+import uiController from './Controllers/UIController'
+
+
+export default class Slider extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            value: props.sliderData.position
+        }
+        this.onSliderChange = this.onSliderChange.bind(this)
+        this.onButtonPress = this.onButtonPress.bind(this)
+    }
+
+    onSliderChange(value) {
+        this.setState({ value }, () => {
+            uiController.onSliderKeepContext(this.props.sliderData.msgID, this.props.sliderAppId)
+            store.dispatch(updateSliderPosition(this.props.sliderAppId, value))
+        })
+    }
+
+    onButtonPress(offset){
+        if(this.state.value + offset > 0 && this.state.value + offset <= this.props.sliderData.numTicks) {
+            this.onSliderChange(this.state.value + offset)
+        }
+    }
+
+    render() {
+        const sliderData = this.props.sliderData
+        let markerColors = {
+            active: this.props.theme ? "#E6E6E6" : "#666666",
+            inactive: this.props.theme ? "#4D4D4D" : "#FFFFFF",
+            selected: this.props.theme ? "#FFFFFF" : "#000000"
+        }
+        
+        return (
+            <div className="slider">
+                <div className="slider-header">
+                    <h1 className="t-small t-medium th-f-color">
+                        {this.props.sliderName}
+                    </h1>
+                    <p className="t-small t-light th-f-color">
+                        {sliderData.header}
+                    </p>
```

Comment:
Might be worth putting some padding between these

Reply:
I also noticed that the text has the opposite weights when compared to an alert (app name and softbuttons have `t-light`, while the alert fields have `t-medium`). Perhaps it should match the alert weights instead?

Reply:
Changed in https://github.com/smartdevicelink/generic_hmi/pull/393/commits/753e34795d8550eebc85d23480f63dbf9b190ef2